docs/courses/skill.md

```diff
@@ -0,0 +1,173 @@
+- `Skill > Thumbnails`: A list of filenames of the thumbnails to be used on the course page to give an idea of the skill. A list of available files can be found on [`apps/web/static/images/`](https://github.com/kantord/LibreLingo/tree/main/apps/web/static/images). The names should be used without extension and without `_tiny` or `_tinier` parts. e.g. `banana2_tinier.jpg` should be written as `banana2`.
+
+**`New words`** has a list of new words that the lesson asks about.
+- `Word`: The word in destination language.
+- `Synonyms`: A list of synonyms of the above word. (optional)
+- `Translation`: Translation of the word in target audience's language.
+- `Also accepted`: A list of synonyms of the `Translation` word above. (optional)
+- `Images`: A list of images for the word that is defined under `Word` or `Translation` above. The image names has to be without extension and without `_tiny` or `_tinier` part.
+
+**`Phrases`** has a list of sentences or phrases that the lession asks about.
+- `Phrase`: The phrase in destination language. e.g. In this case, it's written in Spanish.
+- `Alternative versions`: An alternative version of the above phrase. (optional)
+- `Translation`: Translation of the phrase in target audience's language. e.g. In this case, it's written in English.
+
+**`Mini-dictionary`** has a list of words and meanings. [TODO: Explain where and how it is used]
+- `<destination language>`: A list of words in destination language as key and meaning in target audience's language.
+- `<target audience's language>`: A list of words in target audience's language as key and meaning in destination language.
+
+<a id="creating-new"></a>
+## Creating new skills
+
+Creating new skills is a matter of creating their corresponding YAML file.
+That being said, there are a couple things to keep in mind:
+
+- Skill yaml file should be inside a [module](module.md)'s `skills` directory.
+- Skills should have an unique `Id`. Such identifiers can be generated using
+  [this website](https://www.uuidgenerator.net/version4).
+- Before skills will actually show up in the web app, they need to be added to
+  a [`module.yaml`](module.md#yaml).
+
+<a id="example-edit"></a>
+## Examples of editing a skill
+
+For example, this is how you'd want to add a new alternative version to the
+phrase "La femme dit bonjour":
+
+```diff
+Phrases:
+  - Phrase: La femme dit bonjour
+    Alternative versions:
+      - La femme dit salut
++     - La madame dit salut
+    Translation: The woman says hello
+    Alternative translations:
+      - The woman says hi
+```
+
+Keep in mind that when you add a new phrase, or a new version or translation
+to a phrase, it's not automatically updating to the mini-dictionary. So you'd
+want to add any new words there:
+
+```diff
+Mini-dictionary:
+  French:
+    - dit: says
++   - madame: lady
+    - bonjour:
+      - hello
+      - hi
+    - L'homme: the man
+```
+
+<a id="tips"></a>
+## Tips for creating good skills
```

Comment:
I would maybe move this (and all info about creating the content itself) into a new file :thinking: 

And the title would be something like `How to create good course content`

Reply:
Another tip we could add on that page is

```
Use Words to teach nouns that can be demonstrated easily with pictures, such as "dog", "car", "tree", "city". Do not try to teach verbs, adjectives, etc. using Words. Instead, use them in phrases.
```

Reply:
Great. This is my suggestion with slight changes:

```
- Use words to teach nouns that can be demonstrated easily with pictures, such as "dog", "car", "tree", "city".
- Do not try to teach verbs, adjectives, etc. using words. Instead, use them in phrases.
```

- Lower-cased "word"
- Divided it in 2 points